### PR TITLE
Sound effects

### PR DESCRIPTION
**Pull request by RogierB**

### CODE REVIEW

File: src/Scenes/Game.cpp [hunk 1]
```diff
@@ -239,6 +254,9 @@ void scenes::Game::move_player()
         }
     }
 
+    if (move_up || move_down || move_left || move_right)
+        sound_footstep.play_once();
```
Comment (beschoenen) [inline src/Scenes/Game.cpp]: Brackets

File: src/Scenes/Game.cpp [hunk 2]
```diff
@@ -573,11 +594,18 @@ void scenes::Game::handle_going_through_portal()
...
             level++;
             round = 1;
             hud->set_level(level);
             hud->set_round(round);
             portal_opened = false;
+
+            //Wait for 2 seconds so the portal sound can play fully hearable
+            clock_t start_time = clock();
+            clock_t end_time = 2 * 1000 + start_time;
+            while (clock() != end_time);
```
Comment (beschoenen) [inline src/Scenes/Game.cpp]: Use a ticker, like we use with phase changing.

File: src/defines.h [hunk 3]
```diff
@@ -27,7 +27,7 @@
 
 #define KEYBINDINGS_DIR "Key Bindings"
 
-#define GAME_ROUNDS 10
+#define GAME_ROUNDS 3
```
Comment (beschoenen) [inline src/defines.h]: Put this back

File: src/Scenes/Game.cpp [hunk 4]
```diff
@@ -600,14 +615,11 @@ void scenes::Game::handle_going_through_portal()
...
             portal_opened = false;
-
-            //Wait for 2 seconds so the portal sound can play fully hearable
-            clock_t start_time = clock();
-            clock_t end_time = 2 * 1000 + start_time;
-            while (clock() != end_time);
-
-            Engine::instance()->go_to_scene<WinScreen>();
-
+            if (portal_ticker == -1)
```
Comment (beschoenen) [inline src/Scenes/Game.cpp]: Put an empty line before this if

File: src/defines.h (hunk 3 above)
Comment (RogierB) [reply src/defines.h]: Nee, leef er maar mee

**Pull request merged**